pkg/kf/commands/apps/push_test.go

```diff
@@ -83,7 +94,7 @@ func TestPushCommand(t *testing.T) {
 				"-e", "env2=val2",
 				"--container-registry", "some-reg.io",
 				"--instances", "1",
-				"--path", "testdata/example-app",
+				"--path", "internal/testdata/example-app",
```

Comment:
do we need to move testdata? from my understanding go treats it as a special directory that's supposed to hold the fixtures for the tests in its parent

Reply:
TIL, moved it back